src/routers/product-router.js

```diff
@@ -7,12 +7,16 @@ const {
   validateProductId, validateCreateProduct, validateUpdateProduct
 } = require('../middlewares/validators/product-validator');
 
+const { validateUserId } = require('../middlewares/validators/user-validator');
+const { validateSkipLimit } = require('../middlewares/validators/handlers');
+
 const {
-  getProduct, getProducts, updateProduct, createProduct, uploadImages
+  getProduct, getProducts, getUserProducts, updateProduct, createProduct, uploadImages
 } = require('../controllers/product-controller');
 
 router.get('/', getProducts);
 router.post('/', validateCreateProduct, createProduct);
+router.get('/:userId', validateUserId, validateSkipLimit, getUserProducts);
```

Comment:
this shouldn't be in this router

/users/:userId/products

it should go to user router, but the controller fn should stay in product